app/schema.graphql

```diff
@@ -580,6 +581,32 @@ type Experiment implements Node {
   """Sequence number (1-based) of experiments belonging to the same dataset"""
   sequenceNumber: Int!
   runs(first: Int = 50, last: Int, after: String, before: String): ExperimentRunConnection!
+  annotationSummaries(first: Int = 50, last: Int, after: String, before: String): ExperimentAnnotationSummaryConnection!
+}
+
+type ExperimentAnnotationSummary implements Node {
+  """The Globally Unique ID of this object"""
+  id: GlobalID!
+  annotationName: String!
+  meanScore: Float!
```

Comment:
what will this be if all evals error out?

Reply:
This is an aggregation over annotations, so if the evals error out, the annotation summaries list will be empty.

Reply:
I think this needs to be nullable since it's nullable in the DB

Reply:
addressed in [e8f8910](https://github.com/Arize-ai/phoenix/pull/3549/commits/e8f8910b7a00b7de372d86aa7289b6a58e819650)